lib/index.js

```diff
@@ -102,6 +105,15 @@ module.exports = function (gulp, options) {
     runWithDependencyProcess(runMocha, _.flattenDeep([options.unitTestFiles, options.integrationTestFiles]), cb);
   });
 
+  gulp.task('test-junit', 'Run all the tests with code coverage and output the results as JUNIT. (Specify ', ['lint-junit'], function (cb) {
```

Comment:
Since there is no good way to do per task configuration, a separate task enables us to have a test that outputs junit and one that outputs 'spec' or your reporter of choice. Otherwise, you would have to change the configuration each time.